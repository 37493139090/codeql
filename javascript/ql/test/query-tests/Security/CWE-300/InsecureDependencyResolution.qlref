query: Security/CWE-300/InsecureDependencyResolution.ql
postprocess: utils/test/InlineExpectationsTestQuery.ql
