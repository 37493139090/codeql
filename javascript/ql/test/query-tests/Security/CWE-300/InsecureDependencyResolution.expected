#select
| package.json:6:17:6:40 | "http:/ ... rg/foo" | Dependency downloaded using unencrypted communication channel. |
| package.json:7:17:7:39 | "ftp:// ... rg/foo" | Dependency downloaded using unencrypted communication channel. |
| package.json:12:17:12:40 | "http:/ ... rg/foo" | Dependency downloaded using unencrypted communication channel. |
| package.json:13:17:13:39 | "ftp:// ... rg/foo" | Dependency downloaded using unencrypted communication channel. |
testFailures
| package.json:6:17:6:40 | Dependency downloaded using unencrypted communication channel. | Unexpected result: Alert |
| package.json:7:17:7:39 | Dependency downloaded using unencrypted communication channel. | Unexpected result: Alert |
| package.json:12:17:12:40 | Dependency downloaded using unencrypted communication channel. | Unexpected result: Alert |
| package.json:13:17:13:39 | Dependency downloaded using unencrypted communication channel. | Unexpected result: Alert |
